From 1582de655661622de08e23be13e38f06b1fab325 Mon Sep 17 00:00:00 2001
From: John Poet <jpoet@mythtv.org>
Date: Wed, 11 Jun 2014 16:54:50 -0600
Subject: [PATCH 01/11] Channel scanning: unbreak the cli.  Teardown() frees
 and NULLs sigmonScanner, so testing it to be non-zero
 is always going to fail. Further, if sigmonScanner is
 invalid, then transports will be empty, so that is
 all the test we need.

---
 mythtv/libs/libmythtv/channelscan/channelscanner_cli.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythtv/channelscan/channelscanner_cli.cpp b/mythtv/libs/libmythtv/channelscan/channelscanner_cli.cpp
index 0e323da..2bfbb25 100644
--- a/mythtv/libs/libmythtv/channelscan/channelscanner_cli.cpp
+++ b/mythtv/libs/libmythtv/channelscan/channelscanner_cli.cpp
@@ -74,7 +74,7 @@ void ChannelScannerCLI::HandleEvent(const ScannerEvent *scanEvent)
             QString error = scanEvent->strValue();
             InformUser(error);
         }
-        else if (sigmonScanner && !transports.empty())
+        else if (!transports.empty())
             Process(transports);
 
         done = true;
-- 
1.7.10.2

